core/.env_sample

```diff
@@ -1 +1,2 @@
-MNEMONIC="your mnemonic here"
\ No newline at end of file
+MNEMONIC="your mnemonic here"
+SLACK_WEBHOOK="web hook for your slack app here"
```

Comment:
You might want to include a guide for would-be arbitrageurs for how to create a slack webhook, but perhaps thats outside the scope of this PR

Reply:
It is out of scope for this PR but I will add a comment with a link to the slack docs outlining this process.